samples/csharp_dotnetcore/55.teams-link-unfurling/AdapterWithErrorHandler.cs

```diff
@@ -18,9 +18,10 @@ public AdapterWithErrorHandler(IConfiguration configuration, ILogger<BotFramewor
                 // Log any leaked exception from the application.
                 logger.LogError(exception, $"[OnTurnError] unhandled error : {exception.Message}");
 
-                // Send a message to the user
-                await turnContext.SendActivityAsync("The bot encounted an error or bug.");
-                await turnContext.SendActivityAsync("To continue to run this bot, please fix the bot source code.");
+                // Note: Since this Messaging Extension does not have the messageTeamMembers permission
```

Comment:
This is an open question. Do we want to cut out the commented out lines of this sample? I'm not opposed to having them, but it looks to be inconsistent with other samples. 

The TeamsMessagingExtensionSearch has the lines in, but doesn't have the messageTeamMembers permission in the manifest. 

The TeamsMessagingExtensionAction has the lines commented out, but also does not have the permission in the manifest.